.github/workflows/test.yml

```
@@ -0,0 +1,64 @@
+name: test
+
+on:
+  push:
+    branches:
+      # - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.6"
+          - "2.7"
+          - "3.0"
+          - "3.1"
+          - "3.2"
+          - "3.3"
+        rails:
+          - "5.2"
+          - "6.0"
+          - "6.1"
+          - "7.0"
+        cassandra:
+          - "2.2"
+          - "3.0"
+          - "3.11"
+        exclude:
+          - ruby: "2.6"
+            rails: "7.0"
+          - ruby: "3.0"
+            rails: "5.2"
+          - ruby: "3.1"
+            rails: "5.2"
+          - ruby: "3.2"
+            rails: "5.2"
+          - ruby: "3.3"
+            rails: "5.2"
```

```suggestion
        ruby:
          - 2.7
          - 3.0
          - 3.1
          - 3.2
          - 3.3
        rails:
          - 6.0
          - 6.1
          - 7.0
        cassandra:
          - 2.2
          - 3.0
          - 3.11
```
We can drop 2.6 and 5.2 since we no longer use either and they're well passed EOL

---

Could you also add 7.1 to the rails section? Sorry I missed that. That will also require adding `gemfiles/rails_7.1.gemfile` you can copy rails_7.0.gemfile and update the activemodel dep from 7.0 to 7.1

---

bump ^

---

Added in latest push